jena-tdb1/src/main/java/org/apache/jena/tdb/sys/Names.java

```diff
@@ -21,64 +21,64 @@
 /** Names of things in TDB */
 public class Names
 {
-    public static final String primaryIndexTriples      = "SPO" ; 
+    public static final String primaryIndexTriples      = "SPO" ;
     public static final String[] tripleIndexes          = { primaryIndexTriples, "POS", "OSP" } ;
 
-    public static final String primaryIndexQuads        = "GSPO" ; 
+    public static final String primaryIndexQuads        = "GSPO" ;
     public static final String[] quadIndexes            = { primaryIndexQuads, "GPOS", "GOSP", "POSG", "OSPG", "SPOG"} ;
-    
+
     public static final String primaryIndexPrefix       = "GPU" ;
```

Comment:
Not part of this PR but `GPU` - is this a typo?

Reply:
No :-)

GPU = Graph-Prefix-URI.

It is the prefix storage table.